problems/sumEvenNumbers.js

```diff
@@ -10,7 +10,14 @@
 * returns: 14
 */
 function sumEvenNumbers() {
-
+    let arr = [];
+    let sum = 0;
+    for(let i = 0; i < arr.length; i++) {
+        if(i % 2 === 0) {
+            sum = sum + arr[i];
+        }
+    }
+    return sum;
 }
```

Comment:
You for got to pass in the numbers array  parameter.  The array you created has a length of 0 so the loop you created isn't going to iterate through anything